src/objects/Mesh.js

```diff
@@ -147,17 +147,50 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
-		function checkIntersection( object, material, raycaster, ray, pA, pB, pC, point ) {
+		function checkIntersection( object, material, raycaster, ray, pA, pB, pC, point, faceIndex ) {
 
 			var intersect;
+			var faceMaterial;
+			
+			if ( material.length && faceIndex ) {
+				
+				let groups = object.geometry.groups;
```

Comment:
Can you please use `var` instead of `let`? We are not using `let` and `const` in the project yet.